src/App.js

```diff
@@ -0,0 +1,38 @@
+import { Routes, Route } from "react-router-dom";
+import Home from "./Components/Home";
+import ProductListing from "./Components/Product/ProductListing";
+import Cart from "./Components/Cart/Cart";
+import Wishlist from "./Components/WishList/Wishlist";
+import EmptyWishlist from "./Components/WishList/EmptyWishlist";
+import EmptyCart from "./Components/Cart/EmptyCart";
+import ProductDetail from "./Components/Product/ProductDetail";
+import { useCart } from "./Contexts/data-context";
+import { totalItems } from "./Utils/utils";
+import { ToastContainer } from "react-toastify";
+
+function App() {
+  const { cartItems, wishList } = useCart();
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/product-listing" element={<ProductListing />} />
```

Comment:
The pathname for your routes should be predictable for the end-users and consistent with the other routes. Since you have used the pathname `/products/:productId` for the product detail page, I feel `path="/products"` might be better for the products listing page.